app-antivirus/fangfrisch/fangfrisch-1.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Update and verify unofficial Clam Anti-Virus signatures"
+HOMEPAGE="https://github.com/rseichter/fangfrisch https://pypi.org/project/fangfrisch/"
+SRC_URI="https://github.com/rseichter/fangfrisch/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-python/requests-2.22.0[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.3.11[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+distutils_enable_tests unittest
+
+function python_prepare_all() {
+	sed -i -E -e 's/(SQLAlchemy >= 1.3.1)3/\11/' setup.py || die
+	distutils-r1_python_prepare_all
+}
+
+function python_install_all() {
+	insinto etc
+	newins "${FILESDIR}/conf" ${PN}.conf
+	insinto etc/cron.d
+	newins "${FILESDIR}/cron" ${PN}
```

Comment:
Likewise, you could name this `fangfrisch.cron`.

Reply:
See above.